src/MongoDB/ReadPreference.c

```diff
@@ -105,25 +105,25 @@ PHP_METHOD(ReadPreference, __construct)
 		bson_destroy(tags);
 	}
 
-	if (options && php_array_exists(options, "maxStalenessMS")) {
-		phongo_long maxStalenessMS = php_array_fetchc_long(options, "maxStalenessMS");
-
-		if (maxStalenessMS < 0) {
-			phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Expected maxStalenessMS to be >= 0, %" PHONGO_LONG_FORMAT " given", maxStalenessMS);
-			return;
-		}
-
-		if (maxStalenessMS > INT32_MAX) {
-			phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Expected maxStalenessMS to be <= %" PRId32 ", %" PHONGO_LONG_FORMAT " given", INT32_MAX, maxStalenessMS);
-			return;
-		}
-
-		if (maxStalenessMS > 0 && mode == MONGOC_READ_PRIMARY) {
-			phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "maxStalenessMS may not be used with primary mode");
-			return;
+	if (options && php_array_exists(options, "maxStalenessSeconds")) {
+		phongo_long maxStalenessSeconds = php_array_fetchc_long(options, "maxStalenessSeconds");
+
+		if (maxStalenessSeconds != MONGOC_NO_MAX_STALENESS) {
+			if (maxStalenessSeconds < MONGOC_SMALLEST_MAX_STALENESS_SECONDS) {
+				phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Expected maxStalenessSeconds to be >= %d, %" PHONGO_LONG_FORMAT " given", MONGOC_SMALLEST_MAX_STALENESS_SECONDS, maxStalenessSeconds);
+				return;
+			}
+			if (maxStalenessSeconds > INT32_MAX) {
+				phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Expected maxStalenessSeconds to be <= %" PRId32 ", %" PHONGO_LONG_FORMAT " given", INT32_MAX, maxStalenessSeconds);
+				return;
+			}
+			if (maxStalenessSeconds != MONGOC_NO_MAX_STALENESS && mode == MONGOC_READ_PRIMARY) {
```

Comment:
I just realized this is redundant since we're already within a `maxStalenessSeconds != MONGOC_NO_MAX_STALENESS` conditional. No harm in leaving this in place, though. We can always refactor later.